### ISSUE TITLE
Slow / unresponsive CP when super-table with entries in it.

### ISSUE THREAD

**Issue by martijnvanonz** [engram-design/SuperTable#423]
**Description**
Cp entry whit a super table on it gives a 504. When I remove the super table the cp reacts fast again.

**Steps to reproduce**
1. Create a supertable with table layout one column with a string translatable and one column with a entries
2. create a structure and add the field to the structure.
3. add around 20 entries with all 8 rows in the super-table

**Additional info**
- Plugin version: 2.6.8
- Craft version: Craft Pro 3.6.18
- Multi-site: 2 sites / languages in one group

**Additional context**

**Comment by toyflish** [comment]
same here, behaved fine on
Plugin version: 2.6.7
Craft version: Craft Pro 3.6.13

now entries saving times out after a minute:
Plugin version: 2.6.8
Craft version: Craft Pro 3.7.8

**Comment by toyflish** [comment]
@martijnvanonz do you also have [scout-plugin](https://github.com/studioespresso/craft-scout) installed? 
today I found out it was causing entry-save go up from 5 seconds to 2 minutes even if I passed in empty configuration. Disabling it on plugin-level solved it for me.

**Comment by markdrzy** [comment]
This is also affecting a site I host with a large number of entries. Saving entries results in a 503 due to a MySQL error: "General error: 1205 Lock wait timeout exceeded; try restarting transaction".

Plugin Version: 2.6.8
Craft Version: 3.7.10

Update: I should note that this is a "multisite", with English and Spanish locales.

**Comment by toyflish** [comment]
there is an issue going on on scout plugin https://github.com/studioespresso/craft-scout/issues/205 about this behaviour since craft 3.7

**Comment by timkelty** [comment]
@martijnvanonz make sure you're on the latest (craftcms/cms@3.7.11), we just fixed an index regression issue that affected save performance.

**Comment by sjcallender** [comment]
@markdrzy - Fancy seeing you here! @toyflish and I seem to be having the same issue. Do you have Scout installed too?

**Comment by markdrzy** [comment]
Hey @sjcallender ! I do not have Scout installed, buuuuut I'm extremely interested in what @timkelty just mentioned. I scanned through the 3.7.10 -> 3.7.11 changes and didn't see anything that stood out to me other than:

- Improved the performance of entry queries for installs with several hundred thousand entries or more.

That doesn't sound like it has anything to do with _saving_ entries. I haven't had time to update the site I was battling with to 3.7.11 yet, but that's next on my list. I'll report back with whether or not that takes care of my issue.

**Comment by sjcallender** [comment]
Super Table is not our issue. We still have it even after I disabled it. I believe we're facing this issue with Neo: https://github.com/spicywebau/craft-neo/issues/494

**Comment by turnstylerj** [comment]
Sounds like there will be a fix available for this in 3.7.11, would be great to get this implemented into Super Table. https://github.com/craftcms/cms/issues/9802#issuecomment-914910382

**Comment by engram-design** [comment]
@turnstylerj Will dive into the changes!

**Comment by OwenMelbz** [comment]
Hi,

Is there any progression on this? I think we might be a victim of the slowness as well, when we disable Super table the CP suddenly gets fast again!

**Comment by engram-design** [comment]
I'm been porting over a number of content changes caused by Craft 3.7+ changes, and would be great if you're able to test these out. To get the fixes early, change your `verbb/super-table` requirement in `composer.json` to:

```
"require": {
  "verbb/super-table": "dev-craft-3 as 2.6.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Should be fixed in [2.7.0](https://github.com/verbb/super-table/releases/tag/2.7.0)
